src/scripts.js

```diff
@@ -46,6 +55,17 @@ async function startApp() {
   let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
 }
 
+function buttonHandler(event) {
```

Comment:
Looks good, I think this new layout is going to be a huge benefit for the user. Cleaner data.